crates/bitwarden-core/src/mobile/crypto.rs

```diff
@@ -6,6 +6,8 @@ use bitwarden_crypto::{
 };
 use schemars::JsonSchema;
 use serde::{Deserialize, Serialize};
+#[cfg(feature = "wasm")]
+use {tsify_next::Tsify, wasm_bindgen::prelude::*};
```

Comment:
Huh, I just discovered now thanks to this that you can use a top level `{}` block in a `use` statement, neat!